src/lib/transformGoogleData.js

```diff
@@ -0,0 +1,51 @@
+/* function that transforms google map places results
+to mimic the format of wikipedia results */
+
+/**
+ * Tranform google map places results
+ * to mimic the format wikipedia results
+ *
+ * @param {Object} googleSearchResult The nearby location result that the search term returns
+ *
+ * @return {Object} returns an object matching wikipedia format results
+ */
+
+const transformGooglePlaceData = (googleSearchResult) => {
+  let transformedData = {
+    title: '',
+    thumbnail: {
+      source: '',
+      height: 100,
+      width: 100,
+    },
+    coordinates: [{ lat: '', lon: '' }],
+    description: '',
+    pageid: '',
+  };
+  transformedData.title = googleSearchResult.name;
+  transformedData.thumbnail.source = googleSearchResult.photos
+    ? googleSearchResult.photos[0].getUrl({
+        maxWidth: 100,
+        maxHeight: 100,
+      })
+    : (transformedData.thumbnail = null);
+  transformedData.coordinates[0].lat = googleSearchResult.geometry.location.lat();
+  transformedData.coordinates[0].lon = googleSearchResult.geometry.location.lng();
+  transformedData.description = googleSearchResult.vicinity;
+  transformedData.pageid = googleSearchResult.place_id;
+
```

Comment:
Very cool, glad y'all were able to transform the data here! Love that the infoWindows are still functional even with the google search results